executor/show.go

```diff
@@ -340,6 +343,37 @@ func (e *ShowExec) fetchShowBind() error {
 	return nil
 }
 
+func (e *ShowExec) fetchShowBindingCacheStatus(ctx context.Context) error {
+	exec := e.ctx.(sqlexec.RestrictedSQLExecutor)
+
+	rows, _, err := exec.ExecRestrictedSQL(ctx, nil, fmt.Sprintf("SELECT count(*) FROM mysql.bind_info where status = '%s' or status = '%s';", bindinfo.Enabled, bindinfo.Using))
```

Comment:
Need to add the `Disabled` status binding.

Reply:
The tests for `Dsiabled= status` will be added until the PR https://github.com/pingcap/tidb/pull/32819 is merged.